libs/sdk-backend-bear/src/convertors/fromBackend/DashboardConverter/tests/scheduledEmail.test.ts

```diff
@@ -0,0 +1,115 @@
+// (C) 2022 GoodData Corporation
+
+import { GdcScheduledMail } from "@gooddata/api-model-bear";
+import { convertScheduledMail } from "../scheduledMails";
+
+describe("scheduled mail converter", () => {
+    it("converts scheduled email object", () => {
+        const scheduledMail: GdcScheduledMail.IWrappedScheduledMail = {
+            scheduledMail: {
+                content: {
+                    when: {
+                        recurrency: "recurency string",
+                        startDate: "2022-01-01",
+                        timeZone: "CET",
+                        endDate: "2023-01-01", //?
```

Comment:
what are those commented out question mark here?

Reply:
I forgot it here. It was just marking optional fields